web/src/app/shared/master-and-detail/master-and-detail.component.ts

```diff
@@ -11,7 +11,8 @@ import { MasterAndDetailService } from 'app/services/masterAndDetail/master-and-
 export class MasterAndDetailComponent implements OnInit {
     private classifications = 'classifications';
     private workbaskets = 'workbaskets';
```

Comment:
Use const for constants

Reply:
i don't think you can use constants for class members

Reply:
forgive this comment... but constant can be declared outside of the class. But don't do It, I will do and apply some refactor for that.